src/messages/List/List.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC, Fragment, useEffect, useMemo } from "react";
+import ListMain from "./ListMain";
+import { useMessageContext } from "src/hooks";
+import ListRegular from "./ListRegular";
+import mainclasses from "src/main.module.css";
+import classes from "./List.module.css";
+import { SingleButton } from "src/common/ActionButtons";
+
+const List: FC = () => {
+	const { message, config, action } = useMessageContext();
+	const { payload } = message?.data?._cognigy?._webchat?.message?.attachment || {};
```

Comment:
Use getChannelPayload to extract. See src/messages/TextWithButtons/TextWithButtons.tsx:18

Reply:
done